crates/autopilot/src/driver_model.rs

```diff
@@ -115,6 +115,7 @@ pub mod solve {
         pub app_data: AppDataHash,
         #[serde(flatten)]
         pub signature: Signature,
+        pub protocol_fee: ProtocolFee,
```

Comment:
I feel like this should be a list of ProtocolFee objects, so that we can eventually have multiple fees per order (e.g. a LimitOrder that is coming via a widget integration partner may have a volume based fee for the integration partner and the limit order fee from the protocol).

Also I would call the fee_policy or the like (since unlike user_fee it's not specifying a concrete amount but instead an abstract policy we want the driver to enforce).

Reply:
It should be a list if multiple fees are potentially be applied. Fixed.